components/templates/newRecordSteps.tsx

```diff
@@ -201,7 +192,7 @@ export function NewRecordSummaryStep() {
             <Typography variant="subtitle1">John Smith</Typography>
           </ListItem>
           <ListItem>
-          <ListItemText>Counsellor's notes:</ListItemText>
```

Comment:
ahaha, I think it should be with double LL, if we speak about an attorney

Reply:
you know, it got highlighted by spell checker on VS Code, and when I look for it, looks like it can be written a single or double LL, interesting...